### ISSUE TITLE
This function is obsolete -- use SET-STARTING-HOLE-DIVISOR instead

### ISSUE THREAD

**Issue by binghe** [binghe/axiom#1]
This first line of GCL warning shows up every time when axiom is started.

**Comment by binghe** [comment]
Found this from GCL mailing list:
```
[Gcl-devel] SET-STARTING-HOLE-DIVISOR

From:	 Camm Maguire
Subject: [Gcl-devel] SET-STARTING-HOLE-DIVISOR
Date:	 Thu, 10 Jul 2014 14:41:06 -0400

Greetings!

2.6.10 has a significantly improved memory layout and fixnum handling.
Much work was also done on graceful oom handling encountered by some
acl2 jobs on 'lesser' machines.  The statically compiled maxpage
variable was replaced with a runtime probe of the maximum number of
pages that can be brk'ed at each image startup.  The upshot is that GCL
needed to scale the hole dynamically, so #'si::set-hole-size had to
become a noop.

You can 

1) safely ignore the error
2) remove the set-hole-size call and let gcl handle the memory, or
3) replace the call with si::set-starting-hole-divisor, which will set
the initial hole size as a fraction of the probed maxpages.

This is simply a performance tweak.  A generous hole lowers the starting
gc overhead.  GCL will try to optimize this by itself, but you can
change the initial placement if you want to.

Take care,
-- 
Camm Maguire                                        address@hidden
==========================================================================
"The earth is but one country, and mankind its citizens."  --  Baha'u'llah
```

**Comment by binghe** [comment]
Note also that the above email of GCL maintainer was sent on July 2014.  But this issue always happens with Axiom "Aug 2014" release (the last official binary release), which is built by GCL 2.6.11:
```
This function is obsolete -- use SET-STARTING-HOLE-DIVISOR instead
                        AXIOM Computer Algebra System 
                        Version: Axiom (August 2014)
                Timestamp: Monday July 28, 2014 at 04:12:34 
-----------------------------------------------------------------------------
   Issue )copyright to view copyright notices.
   Issue )summary for a summary of useful system commands.
   Issue )quit to leave AXIOM and return to shell.
   Visit http://axiom-developer.org for more information
-----------------------------------------------------------------------------
 
   Re-reading interp.daase
   Re-reading operation.daase
   Re-reading category.daase
   Re-reading browse.daase
(1) -> 
(1) -> )lisp (lisp-implementation-version)
(1) -> 
Value = "GCL 2.6.11"
```

According to GCL official web site [1], GCL 2.6.10 was released on 20131113, and GCL 2.6.11 on 20140906. Axiom should be using a pre-release version of GCL 2.6.11 when making its public release - maybe it's identical with the 2.6.11 final release.

[1] https://www.gnu.org/software/gcl/

**Comment by binghe** [comment]
Fixed in [6d734de](https://github.com/binghe/axiom-cas/commit/6d734de631263a09858b89bd1d6d22dfa6410fd2)
